src/app/assets/SunIcon.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+
+export default function SunIcon(): JSX.Element {
+  return (
+    <svg
+      width="42"
+      height="42"
+      viewBox="0 0 42 42"
+      fill="none"
+      xmlns="http://www.w3.org/2000/svg"
```

Comment:
You don't want to pass props?

Reply:
I thought about it but i think i don´t need properties in this component. The header is static. Or have you any ideas what i could pass? :)